node/tests/ServerModules.test.ts

```diff
@@ -2499,7 +2501,80 @@ describe("Server Module Tests", () => {
         });
 
         it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
-            "FT.AGGREGATE ft.aggregate",
+            "FT.INFO ft.info",
```

Comment:
moved from below

Reply:
I don't know how jest applies test filter (case sensitive or not) so I duplicated the test names